src/der.rs

```diff
@@ -178,10 +191,13 @@ pub fn positive_integer<'a>(input: &mut untrusted::Reader<'a>)
 
 
 #[cfg(test)]
-mod tests {
+pub mod tests {
+    use std;
+    use std::io::BufRead;
     use error;
     use super::*;
     use untrusted;
+    use rustc_serialize::base64::FromBase64;
```

Comment:
I don't care. It looks like it's not done consistently in the existing code, which is unfortunate, but probably inevitable unless/until we have a tool enforce it.